prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts

```diff
@@ -62,7 +65,11 @@ export class DashboardComponent implements OnInit {
   }
 
   public onLogout() {
-    this.authService.logout(this.config.loginRedirectUrl);
+    if (this.authService.isAdmin()) {
+      this.authService.logout(this.config.loginRedirectUrl + '/' + AuthRoutes.ADMIN);
```

Comment:
Though there isn't anything wrong with concatenating using + operator in ES6 the backtick can be used to interpolate variables like ``${this.config.loginRedirectUrl}/${AuthRoutes.ADMIN}``

Reply:
Yes thats better, thanks